indico/modules/events/editing/operations.py

```diff
@@ -204,8 +214,25 @@ def create_submitter_revision(prev_revision, user, files):
     return new_revision
 
 
+@memoize
```

Comment:
I don't think memoizing this is a good idea. The data is stored globally so it's really meant for things that should be memoized forever, not just in the context of a request. We have `@memoize_request` for that.